### ISSUE TITLE
SeAT Dev: command `seat:admin:login broken`

### ISSUE THREAD

**Issue by herpaderpaldent** [eveseat/seat#324]
* **Problem**: i must go to a birthday party can't tackle this myself:

```
/var/www/seat # php artisan seat:admin:login
SeAT Admin Login URL Generator
User 'admin' does not exist. It will be created.

In Connection.php line 664:

  SQLSTATE[42S22]: Column not found: 1054 Unknown column 'main_character_id'
  in 'field list' (SQL: insert into `groups` (`main_character_id`, `updated_a
  t`, `created_at`) values (1, 2018-05-19 11:29:17, 2018-05-19 11:29:17))


In PDOConnection.php line 82:

  SQLSTATE[42S22]: Column not found: 1054 Unknown column 'main_character_id'
  in 'field list'


In PDOConnection.php line 80:

  SQLSTATE[42S22]: Column not found: 1054 Unknown column 'main_character_id'
  in 'field list'


```

* **Expected**: `generate a admin login url
* **Logs / Screenshots / Proof**: _The more information you give the better!_
* **Version Info**:  75eb7989082aa8e2a53b489ef3e08a2d78a1e396

**Comment by solamance** [comment]
I ended up getting past this error ( and the one that follows) by creating the missing fields in the respective tables (main_character_id in seats.groups and email in seats.users) - however;

I discovered while tinkering with fixing this, that you need to spin up your instance, log in as your own user, then edit the tables, then create the admin user, or all the API calls broke. That said, it's possible I may have gotten my main character locked down by repeatedly troubleshooting this and spamming ESI unintentionally (I feel rather silly).

Either way - once those two fields are added, I was able to create the admin login url.

**Comment by warlof** [comment]
about the admin:login error upper, it will be addressed by https://github.com/eveseat/console/pull/24

it's related to those commit
https://github.com/eveseat/web/commit/910e4d903b7eb06887dfc682dbaf446c98691d2d
https://github.com/eveseat/web/commit/880eaa7b26b27ac8fc5be7fb3dafcf5f0d2c4804
https://github.com/eveseat/web/pull/169/commits/a0c15b4ed76b90161c1297065527b3f0b6d1dd9b

**Comment by solamance** [comment]
Awesome! I figured :)

**Comment by herpaderpaldent** [comment]
Thank you warlof! ....

**Comment by leonjza** [comment]
Fixed in eveseat/console 3.0.0-beta7
